README.md

```diff
@@ -20,6 +20,7 @@ less used and can keep the defaults
 - `DOCKER_GROUP` the docker group name, should be same as the docker's host group (defaults to `docker`)
 - `DOCKER_SOCKET` the docker socket location (default is `/var/run/docker.sock`)
 - `JAVA_OPTS` pass java options to the `agent.jar` process (default is not set)
+- `JENKINS_AGENT_CONNECTION_MODE` the connection mode to use to connect to the jenkins's master (default `-http`). If jenkins is running behind a reverse proxy it is advisable to use `-webSocket` to avoid connection problems (for more details see https://www.jenkins.io/doc/book/managing/cli/)
```

Comment:
I think just explaining what the option does is enough. The **why** to use this option, I assume people who needs it will already understand. Wdyt?

```suggestion
- `JENKINS_AGENT_CONNECTION_MODE` the connection mode to use to connect to the jenkins's master (defaults to `-http`)
```

Reply:
Sounds good to me.